internal/server/node/golden/get_triples/place.json

```diff
@@ -735,9 +735,8 @@
     },
     {
       "subjectId": "ipedsId/242626",
-      "subjectName": "Inter American University of Puerto Rico-Aguadilla",
       "subjectTypes": [
```

Comment:
This is a regression introduced in cl/447061823 that causes some core properties (name, typeOf, containedInPlace) of  certain types that were newly added to the Place table (like CollegeOrUniversity, School) to get filtered out because those provenances are not considered as "trusted".  I'll prepare a fix... but we might need to rebuild infrequent and update the version.

Sorry about that...

Reply:
The fix is in cl/450847262 ... I'll fire off cache builds now

Reply:
Could you try using: frequent_2022_05_26_01_19_55, infrequent_2022_05_25_10_19_22

And see how the diffs look now? Thanks!

(The BQ build is still running, should be an hour or so, but that shouldn't affect the diffs much)

Reply:
updated!

Reply:
Can you sync the PR to HEAD to get all latest mixer changes.

Reply:
Sorry, yes, just did